Box.V2/Managers/BoxFilesManager.cs

```diff
@@ -1326,6 +1327,41 @@ public async Task<BoxRepresentationCollection<BoxRepresentation>> GetRepresentat
             return response.ResponseObject.Representations;
         }
 
+        /// <summary>
+        /// Creates a zip and downloads it to a given Stream.
+        /// </summary>
+        /// <param name="zipRequest">Object of type BoxZipRequest that contains name and items.</param>
+        /// <param name="output">The stream to where the zip file will be written.</param>
+        /// <returns>The status of the download.</returns>
+        /// </summary>
+        public async Task<BoxZipDownloadStatus> DownloadZip(BoxZipRequest zipRequest, Stream output)
```

Comment:
Move to a new class "BoxZip" to match the [Box Java SDK](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxZip.java#L16)?